force-app/main/default/triggers/ContactTrigger.trigger

```diff
@@ -19,10 +19,21 @@
 trigger ContactTrigger on Contact(before insert) {
 	// When a contact is inserted
 	// if DummyJSON_Id__c is null, generate a random number between 0 and 100 and set this as the contact's DummyJSON_Id__c value
+	if (Trigger.isBefore && Trigger.isInsert){
```

Comment:
You are calling each method in the correct context, but once you update your ContactTriggerHandler to extend the methods and properties from the TriggerHandler you can remove these conditional blocks and just have a single line that executes the handler:
```
trigger ContactTrigger on Contact(before insert, after insert, after update) {
	new ContactTriggerHandler().run();
}
```